src/shared/constants/legal-requirements/legal-requirements.constants.ts

```diff
@@ -10,6 +10,8 @@ export enum legalRequirementsLabels {
   eco = 'Eco (control and planning)',
   vehicleChecks = 'Show me / Tell me',
   uncoupleRecouple = 'Uncouple / Recouple',
+  busStop1 = 'BS (bus stop)',
+  busStop2 = 'BS (bus stop',
```

Comment:
the difference between these two is the missing parenthesise?

Reply:
they should be the same. Nice spot.